config.yaml.sample

```diff
@@ -21,6 +21,9 @@ grafeas:
     cafile:
     keyfile:
     certfile:
+    cors_allowed_origins:
+       - "https://some.example.tld"
```

Comment:
I prefer the sample config to be the same as the default config. This means that the list of allowed origins should be empty by default. So either remove the example urls or move them to a comment.